addons/l10n_latam_invoice_document/models/account_move_line.py

```diff
@@ -28,17 +28,20 @@ def compute_l10n_latam_prices_and_taxes(self):
             included_taxes = \
                 invoice.l10n_latam_document_type_id and invoice.l10n_latam_document_type_id._filter_taxes_included(
                     line.tax_ids)
+            # For the unit price, we need the number rounded based on the product price precision.
+            # The method compute_all uses the accuracy of the currency so, we multiply and divide for 10^(decimal accuracy of product price) to get the price correctly rounded.
+            price_digits = 10**self.env['decimal.precision'].precision_get('Product Price')
             if not included_taxes:
                 price_unit = line.tax_ids.with_context(round=False, force_sign=invoice._get_tax_force_sign()).compute_all(
-                    line.price_unit, invoice.currency_id, 1.0, line.product_id, invoice.partner_id)
-                l10n_latam_price_unit = price_unit['total_excluded']
+                    line.price_unit, invoice.currency_id, price_digits, line.product_id, invoice.partner_id)
```

Comment:
```suggestion
                    line.price_unit * price_digits, invoice.currency_id, 1.0, line.product_id, invoice.partner_id)
```

Changing the quantity might have weird side effects in corner cases, and it is less readable.

Reply:
Oh yeah, I agree, thanks for the suggestion